api/index.py

```
@@ -304,20 +340,21 @@ def chat():
     repo = data.get('repo')
     if repo is None:
-        return jsonify({'status': f'Missing mandatory parameter "repo"'}), 400
+        return jsonify({'status': 'Missing mandatory parameter "repo"'}), 400
 
     # Get optional 'label' and 'relation' parameters
     msg = data.get('msg')
     if msg is None:
-        return jsonify({'status': f'Missing mandatory parameter "msg"'}), 400
+        return jsonify({'status': 'Missing mandatory parameter "msg"'}), 400
 
     answer = ask(repo, msg)
```

Ruff (0.8.2)

`ask` may be undefined, or defined from star imports

(F405)